lib/literal/enum.rb

```diff
@@ -199,7 +199,14 @@ def name
 	end
 
 	alias_method :inspect, :name
-	alias_method :to_s, :name
+
+	def to_s
+		self.class.names[self].name.underscore.humanize
```

Comment:
Neither `underscore` nor `humanize` are available in the Ruby standard library. These are patched in by ActiveSupport which is unfortunately included in our test environment in order to test integration with Rails.

I think this `to_s` method should just be the short name of the constant as a string (but not fully qualified). If you’re using rails, you could implement a `humanize` method yourself.